zenodo_accessrequests/views/requests.py

```diff
@@ -156,7 +156,8 @@ def confirm(pid, record, template, **kwargs):
 
     # Confirm email address.
     if r.status != RequestStatus.EMAIL_VALIDATION:
-        abort(404)
+        error_message = "The status of your access request is: " + r.status.value + "."
```

Comment:
- There should be some follow-up information suggested to the user, so they can know what to expect. I.e. if the request is `RequestStatus.PENDING` it should say something like 'Your email address has been verified. and the access request is now pending a decision from the record owner.'
- If it's accepted or rejected, it should say "The request has already been {status}. Check your mail inbox for the record owner's response."
- Also formatting style (see comment/suggestion below)